clientv3/integration/watch_test.go

```diff
@@ -620,11 +620,13 @@ func TestWatchEventType(t *testing.T) {
 		isModify: true,
 	}, {
 		et: clientv3.EventTypeDelete,
+		isDelete: true,
```

Comment:
Doesn't `isDelete` need to be a part of struct here, https://github.com/coreos/etcd/blob/5639bdfd16ae973142b4d7285a22628de1d6c37a/clientv3/integration/watch_test.go#L611 ?